packages/account/src/Components/form-field-info/__tests__/form-field.info.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { FormFieldInfo } from '../form-field-info';
+import userEvent from '@testing-library/user-event';
+
+describe('FormFieldInfo', () => {
+    it('should render the component', () => {
+        render(<FormFieldInfo message='Info content' />);
+        const popover = screen.getByTestId('dt_form-field-info__popover');
+        expect(popover).toBeInTheDocument();
+    });
+
+    it('should toggle popover on click', () => {
+        render(<FormFieldInfo message='Info content' />);
+        const message = screen.queryByText('Info content');
+        expect(message).not.toBeInTheDocument();
+
+        const popover = screen.getByTestId('dt_form-field-info__popover');
+
+        userEvent.click(popover);
+
+        const content = screen.getByText('Info content');
+        expect(content).toBeInTheDocument();
+    });
+
+    it('should close popover when clicking outside', () => {
+        render(<FormFieldInfo message='Info content' />);
+        const message = screen.queryByText('Info content');
+        expect(message).not.toBeInTheDocument();
+
+        const popover = screen.getByTestId('dt_form-field-info__popover');
+
+        userEvent.click(popover);
+
+        const content = screen.getByText('Info content');
+
```

Comment:
@shahzaib-deriv just a nit-pick :) Some line breaks here seem to be extra, and it's better to put all constants(variables) together, then action (click) -> assertion (expect), and another action (click) -> assertion (expect)

Reply:
There's actually no set of rule for this, as it's kinda personal preference, however I'll update this 👌